src/app/routes/expenses.tsx

```
@@ -1,47 +1,72 @@
+        <MonthSelector period={period} onSetPeriod={setPeriod} />
+        <div className='flex flex-col mt-4'>
+          <p className='text-[15px] text-[#555] font-semibold'>총 소비 내역</p>
+          <p className='text-[22px] text-[#222] font-semibold mt-1'>
+            {totalAmount.toLocaleString('ko-KR')}원
+          </p>
         </div>
-        <DailyExpenseList dailyExpenses={dailyExpenses} />
-      </main>
+      </div>
+      <DailyExpenseList dailyExpenses={dailyExpenses} onScroll={handleScroll} />
+      <AddExpenseButton onClick={() => {}} />
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)